### ISSUE TITLE
Sharing rule diff not detected

### ISSUE THREAD

**Issue by alex9138** [scolladon/sfdx-git-delta#29]
Hello @scolladon,

Repro step:
Make a diff in AccountSharingRules  meta file

Result:
sgd didn't ouput any change

Expected Result:
package.xml contains XXXsharingrules meta file and output force-app folder too

SGD version 3.1.0

**SGD output:**
<pre>
alex@ubuntu:~/Documents/dentmaster$ sgd -d -t HEAD -f b96acca834a80540c108d9d2f19b9cc2de84380d -r . -o output
check output
alex@ubuntu:~/Documents/dentmaster$ cat output/package/package.xml 
</pre>
`<?xml version="1.0" encoding="UTF-8"?>
<Package xmlns="http://soap.sforce.com/2006/04/metadata">
    <version>48.0</version>`

**Git ouput:**
<pre>
alex@ubuntu:~/Documents/dentmaster$ git diff HEAD..b96acca834a80540c108d9d2f19b9cc2de84380d
diff --git a/force-app/main/default/sharingRules/Account.sharingRules-meta.xml b/force-app/main/default/sharingRules/Account.sharingRules-meta.xml
index 9234e879..b3dca45f 100644
--- a/force-app/main/default/sharingRules/Account.sharingRules-meta.xml
+++ b/force-app/main/default/sharingRules/Account.sharingRules-meta.xml
@@ -1,4 +1,5 @@
-<?xml version="1.0" encoding="utf-8"?><SharingRules xmlns="http://soap.sforce.com/2006/04/metadata">
+<?xml version="1.0" encoding="UTF-8"?>
+<SharingRules xmlns="http://soap.sforce.com/2006/04/metadata">
     <sharingCriteriaRules>
         <fullName>DWIN2_AccountRH_Edit</fullName>
         <accessLevel>Edit</accessLevel>
@@ -14,7 +15,7 @@
         <criteriaItems>
             <field>RecordTypeId</field>
             <operation>equals</operation>
-            <value>RH</value>
+            <value>RH (Obsolète),RH</value>
         </criteriaItems>
     </sharingCriteriaRules>
     <sharingCriteriaRules>
@@ -33,7 +34,7 @@
         <criteriaItems>
             <field>RecordTypeId</field>
             <operation>notEqual</operation>
-            <value>RH</value>
+            <value>RH (Obsolète),RH</value>
         </criteriaItems>
     </sharingCriteriaRules>
     <sharingCriteriaRules>
@@ -52,7 +53,7 @@
         <criteriaItems>
             <field>RecordTypeId</field>
             <operation>notEqual</operation>
-            <value>RH,PARTICULIER</value>
+            <value>RH (Obsolète),RH,PARTICULIER</value>
         </criteriaItems>
     </sharingCriteriaRules>
     <sharingCriteriaRules>
@@ -71,7 +72,7 @@
         <criteriaItems>
             <field>RecordTypeId</field>
             <operation>equals</operation>
-            <value>RH</value>
+            <value>RH (Obsolète),RH</value>
         </criteriaItems>
     </sharingCriteriaRules>
     <sharingCriteriaRules>
@@ -89,7 +90,25 @@
         <criteriaItems>
             <field>RecordTypeId</field>
             <operation>notEqual</operation>
-            <value>RH</value>
+            <value>RH (Obsolète),RH</value>
+        </criteriaItems>
+    </sharingCriteriaRules>
+    <sharingCriteriaRules>
+        <fullName>DWIN2_Account_Client_ReadCommunityEmployee</fullName>
+        <accessLevel>Read</accessLevel>
+        <accountSettings>
+            <caseAccessLevel>None</caseAccessLevel>
+            <contactAccessLevel>Read</contactAccessLevel>
+            <opportunityAccessLevel>None</opportunityAccessLevel>
+        </accountSettings>
+        <label>DWIN2_Account_Client_ReadCommunityEmployee</label>
+        <sharedTo>
+            <group>DWIN2_PartnerCommunity_ROAD</group>
+        </sharedTo>
+        <criteriaItems>
+            <field>RecordTypeId</field>
+            <operation>notEqual</operation>
+            <value>RH (Obsolète),RH</value>
         </criteriaItems>
     </sharingCriteriaRules>
     <sharingCriteriaRules>
@@ -109,7 +128,7 @@
         <criteriaItems>
             <field>RecordTypeId</field>
             <operation>notEqual</operation>
-            <value>RH</value>
+            <value>RH (Obsolète),RH</value>
         </criteriaItems>
     </sharingCriteriaRules>
     <sharingCriteriaRules>
alex@ubuntu:~/Documents/dentmaster$ git diff HEAD..b96acca834a80540c108d9d2f19b9cc2de84380d --stat
 force-app/main/default/sharingRules/Account.sharingRules-meta.xml | 33 ++++++++++++++++++++++++++-------
 1 file changed, 26 insertions(+), 7 deletions(-)
</pre>

**Comment by scolladon** [comment]
Hi @alex9138 thanks for raising this issue !

Could you send the full content of the Account.sharingRules file ?
It seems to be changes not in the xml right ?

Seb

**Comment by alex9138** [comment]
@scolladon that's the xml interpretation on the post ;)

<?xml version="1.0" encoding="utf-8"?><SharingRules xmlns="http://soap.sforce.com/2006/04/metadata">
    <sharingCriteriaRules>
        <fullName>DWIN2_AccountRH_Edit</fullName>
        <accessLevel>Edit</accessLevel>
        <accountSettings>
            <caseAccessLevel>Read</caseAccessLevel>
            <contactAccessLevel>Edit</contactAccessLevel>
            <opportunityAccessLevel>Read</opportunityAccessLevel>
        </accountSettings>
        <label>DWIN2_AccountRH_Edit</label>
        <sharedTo>
            <roleAndSubordinatesInternal>DWIN2_RH_Rsp</roleAndSubordinatesInternal>
        </sharedTo>
        <criteriaItems>
            <field>RecordTypeId</field>
            <operation>equals</operation>
            <value>RH</value>
        </criteriaItems>
    </sharingCriteriaRules>
    <sharingCriteriaRules>
        <fullName>DWIN2_Account_ADV_Edit</fullName>
        <accessLevel>Edit</accessLevel>
        <accountSettings>
            <caseAccessLevel>Read</caseAccessLevel>
            <contactAccessLevel>Edit</contactAccessLevel>
            <opportunityAccessLevel>Read</opportunityAccessLevel>
        </accountSettings>
        <description>ADV and SC can edit all customer account</description>
        <label>DWIN2_Account_ADV_Edit</label>
        <sharedTo>
            <group>DWIN2_ADV</group>
        </sharedTo>
        <criteriaItems>
            <field>RecordTypeId</field>
            <operation>notEqual</operation>
            <value>RH</value>
        </criteriaItems>
    </sharingCriteriaRules>
    <sharingCriteriaRules>
        <fullName>DWIN2_Account_Client_Read</fullName>
        <accessLevel>Edit</accessLevel>
        <accountSettings>
            <caseAccessLevel>Read</caseAccessLevel>
            <contactAccessLevel>Edit</contactAccessLevel>
            <opportunityAccessLevel>Read</opportunityAccessLevel>
        </accountSettings>
        <description>Tous les comptes clients sont visible par la branche commerciale</description>
        <label>DWIN2_Account_Client_Read</label>
        <sharedTo>
            <roleAndSubordinatesInternal>DWIN2_Direction</roleAndSubordinatesInternal>
        </sharedTo>
        <criteriaItems>
            <field>RecordTypeId</field>
            <operation>notEqual</operation>
            <value>RH,PARTICULIER</value>
        </criteriaItems>
    </sharingCriteriaRules>
    <sharingCriteriaRules>
        <fullName>DWIN2_Account_RH_Read</fullName>
        <accessLevel>Read</accessLevel>
        <accountSettings>
            <caseAccessLevel>None</caseAccessLevel>
            <contactAccessLevel>Read</contactAccessLevel>
            <opportunityAccessLevel>None</opportunityAccessLevel>
        </accountSettings>
        <description>Can view collaborator</description>
        <label>DWIN2_Account_RH_Read</label>
        <sharedTo>
            <role>DWIN2_Direction</role>
        </sharedTo>
        <criteriaItems>
            <field>RecordTypeId</field>
            <operation>equals</operation>
            <value>RH</value>
        </criteriaItems>
    </sharingCriteriaRules>
    <sharingCriteriaRules>
        <fullName>DWIN2_Account_SC_Edit</fullName>
        <accessLevel>Edit</accessLevel>
        <accountSettings>
            <caseAccessLevel>None</caseAccessLevel>
            <contactAccessLevel>Edit</contactAccessLevel>
            <opportunityAccessLevel>None</opportunityAccessLevel>
        </accountSettings>
        <label>DWIN2_Account_SC_Edit</label>
        <sharedTo>
            <group>DWIN2_ServiceClientGroup</group>
        </sharedTo>
        <criteriaItems>
            <field>RecordTypeId</field>
            <operation>notEqual</operation>
            <value>RH</value>
        </criteriaItems>
    </sharingCriteriaRules>
    <sharingCriteriaRules>
        <fullName>DWIN2_SR_Account_Contact_Create</fullName>
        <accessLevel>Edit</accessLevel>
        <accountSettings>
            <caseAccessLevel>None</caseAccessLevel>
            <contactAccessLevel>Edit</contactAccessLevel>
            <opportunityAccessLevel>None</opportunityAccessLevel>
        </accountSettings>
        <description>Can create contact commercial</description>
        <label>DWIN2_SR_Account_Contact_Create</label>
        <sharedTo>
            <group>DWIN2_PartnerCommunity_ROAD</group>
        </sharedTo>
        <booleanFilter>1</booleanFilter>
        <criteriaItems>
            <field>RecordTypeId</field>
            <operation>notEqual</operation>
            <value>RH</value>
        </criteriaItems>
    </sharingCriteriaRules>
    <sharingCriteriaRules>
        <fullName>DWIN2_SR_Account_PersonAccount_Create</fullName>
        <accessLevel>Edit</accessLevel>
        <accountSettings>
            <caseAccessLevel>None</caseAccessLevel>
            <contactAccessLevel>Edit</contactAccessLevel>
            <opportunityAccessLevel>None</opportunityAccessLevel>
        </accountSettings>
        <description>Partner user can create person account</description>
        <label>DWIN2_SR_Account_PersonAccount_Create</label>
        <sharedTo>
            <group>DWIN2_PartnerCommunity_ROAD</group>
        </sharedTo>
        <criteriaItems>
            <field>RecordTypeId</field>
            <operation>equals</operation>
            <value>PARTICULIER</value>
        </criteriaItems>
    </sharingCriteriaRules>
</SharingRules>

**Comment by scolladon** [comment]
Could you share also the diff without xml formatting ?

**Comment by alex9138** [comment]
Was not able to find a way to kick markdown interpreter... anything to
share? ^^

**Comment by scolladon** [comment]
Even if you use code block ?

**Comment by alex9138** [comment]
Alright:

```
@@ -1,4 +1,5 @@
-<?xml version="1.0" encoding="utf-8"?><SharingRules xmlns="http://soap.sforce.com/2006/04/metadata">
+<?xml version="1.0" encoding="UTF-8"?>
+<SharingRules xmlns="http://soap.sforce.com/2006/04/metadata">
     <sharingCriteriaRules>
         <fullName>DWIN2_AccountRH_Edit</fullName>
         <accessLevel>Edit</accessLevel>
@@ -14,7 +15,7 @@
         <criteriaItems>
             <field>RecordTypeId</field>
             <operation>equals</operation>
-            <value>RH</value>
+            <value>RH (Obsolète),RH</value>
         </criteriaItems>
     </sharingCriteriaRules>
     <sharingCriteriaRules>
@@ -33,7 +34,7 @@
         <criteriaItems>
             <field>RecordTypeId</field>
             <operation>notEqual</operation>
-            <value>RH</value>
+            <value>RH (Obsolète),RH</value>
         </criteriaItems>
     </sharingCriteriaRules>
     <sharingCriteriaRules>
@@ -52,7 +53,7 @@
         <criteriaItems>
             <field>RecordTypeId</field>
             <operation>notEqual</operation>
-            <value>RH,PARTICULIER</value>
+            <value>RH (Obsolète),RH,PARTICULIER</value>
         </criteriaItems>
     </sharingCriteriaRules>
     <sharingCriteriaRules>
@@ -71,7 +72,7 @@
         <criteriaItems>
             <field>RecordTypeId</field>
             <operation>equals</operation>
-            <value>RH</value>
+            <value>RH (Obsolète),RH</value>
         </criteriaItems>
     </sharingCriteriaRules>
     <sharingCriteriaRules>
@@ -89,7 +90,25 @@
         <criteriaItems>
             <field>RecordTypeId</field>
             <operation>notEqual</operation>
-            <value>RH</value>
+            <value>RH (Obsolète),RH</value>
+        </criteriaItems>
+    </sharingCriteriaRules>
+    <sharingCriteriaRules>
+        <fullName>DWIN2_Account_Client_ReadCommunityEmployee</fullName>
+        <accessLevel>Read</accessLevel>
+        <accountSettings>
+            <caseAccessLevel>None</caseAccessLevel>
+            <contactAccessLevel>Read</contactAccessLevel>
+            <opportunityAccessLevel>None</opportunityAccessLevel>
+        </accountSettings>
+        <label>DWIN2_Account_Client_ReadCommunityEmployee</label>
+        <sharedTo>
+            <group>DWIN2_PartnerCommunity_ROAD</group>
+        </sharedTo>
+        <criteriaItems>
+            <field>RecordTypeId</field>
+            <operation>notEqual</operation>
+            <value>RH (Obsolète),RH</value>
         </criteriaItems>
     </sharingCriteriaRules>
     <sharingCriteriaRules>
@@ -109,7 +128,7 @@
         <criteriaItems>
             <field>RecordTypeId</field>
             <operation>notEqual</operation>
-            <value>RH</value>
+            <value>RH (Obsolète),RH</value>
         </criteriaItems>
     </sharingCriteriaRules>
     <sharingCriteriaRules>
```

**Comment by alex9138** [comment]
@scolladon I've also updated the full file as it was interpreted so there was missing lines at the top

**Comment by scolladon** [comment]
Thanks @alex9138 I am able to reproduce locally

**Comment by scolladon** [comment]
@alex9138 could you beta test the #30 branch please ?
It works locally for me and the PR is submitted but I would like your test result also :)

**Comment by alex9138** [comment]
Hello @scolladon that's a big pull request! I'm trying to review it by the way..

On my side I have no package.xml in package/ directory (and no package directory at all) so it can be a problem for those who are using manifest to deploy.

But force-app/.... contains the AccountSharing...xml so that's part is okay for me.


Below my full trace :

```
alex@ubuntu:~/Documents/sgd$ git clean -xdf .
Removing node_modules/
alex@ubuntu:~/Documents/sgd$ git branch
  feature/experience-bundle
* fix/inFile-delta-generation
  master
alex@ubuntu:~/Documents/sgd$ git reset --hard origin/fix/inFile-delta-generation
HEAD is now at 93aa864 Huge commit to fix inFile and asynchronize code
alex@ubuntu:~/Documents/sgd$ npm install

> husky@3.0.3 install /home/alex/Documents/sgd/node_modules/husky
> node husky install

husky > Setting up git hooks
husky > Done

> core-js@3.1.4 postinstall /home/alex/Documents/sgd/node_modules/core-js
> node scripts/postinstall || echo "ignore"

Thank you for using core-js ( https://github.com/zloirock/core-js ) for polyfilling JavaScript standard library!

The project needs your help! Please consider supporting of core-js on Open Collective or Patreon: 
> https://opencollective.com/core-js 
> https://www.patreon.com/zloirock 

Also, the author of core-js ( https://github.com/zloirock ) is looking for a good job -)


> fast-xml-parser@3.17.4 postinstall /home/alex/Documents/sgd/node_modules/fast-xml-parser
> node tasks/postinstall.js || exit 0

Love fast-xml-parser? Check https://amitkumargupta.work for more projects and contribution.


> husky@3.0.3 postinstall /home/alex/Documents/sgd/node_modules/husky
> opencollective-postinstall || true

Thank you for using husky!
If you rely on this package, please consider supporting our open collective:
> https://opencollective.com/husky/donate

npm WARN optional SKIPPING OPTIONAL DEPENDENCY: fsevents@2.1.3 (node_modules/fsevents):
npm WARN notsup SKIPPING OPTIONAL DEPENDENCY: Unsupported platform for fsevents@2.1.3: wanted {"os":"darwin","arch":"any"} (current: {"os":"linux","arch":"x64"})

added 920 packages from 494 contributors and audited 921 packages in 18.074s

15 packages are looking for funding
  run `npm fund` for details

found 6 vulnerabilities (4 low, 2 moderate)
  run `npm audit fix` to fix them, or `npm audit` for details


   ╭────────────────────────────────────────────────────────────────╮
   │                                                                │
   │      New patch version of npm available! 6.14.4 → 6.14.5       │
   │   Changelog: https://github.com/npm/cli/releases/tag/v6.14.5   │
   │               Run npm install -g npm to update!                │
   │                                                                │
   ╰────────────────────────────────────────────────────────────────╯

alex@ubuntu:~/Documents/sgd$ cd ../dentmaster/
alex@ubuntu:~/Documents/dentmaster$ git clean -xdf .
Removing output/
alex@ubuntu:~/Documents/dentmaster$ git reset --hard HEAD
HEAD is now at f5446dc0 Merged PR 81: #129 remove old record type RH
alex@ubuntu:~/Documents/dentmaster$ mkdir output
alex@ubuntu:~/Documents/dentmaster$ ../sgd/bin/cli -d -t HEAD -f b96acca834a80540c108d9d2f19b9cc2de84380d -r . -o output
bash: ../sgd/bin/cli: Permission denied
alex@ubuntu:~/Documents/dentmaster$ chmod +x ../
dentmaster/ sgd/        
alex@ubuntu:~/Documents/dentmaster$ chmod +x ../
dentmaster/ sgd/        
alex@ubuntu:~/Documents/dentmaster$ chmod +x ../sgd/bin/cli 
config/                       force-app/                    .gitattributes                MODIFY TO RESTART CI JOB.txt  README.md                     sfdx-project.json
custom-apex-rules.xml         .forceignore                  .gitignore                    output/                       sample-data/                  
deployment_script.sh          .git/                         manifest/                     pipeline/                     scripts/                      
alex@ubuntu:~/Documents/dentmaster$ chmod +x ../sgd/bin/cli 
alex@ubuntu:~/Documents/dentmaster$ ../sgd/bin/cli -d -t HEAD -f b96acca834a80540c108d9d2f19b9cc2de84380d -r . -o output
check output
alex@ubuntu:~/Documents/dentmaster$ ls -la  output/
total 16
drwxrwxr-x  4 alex alex 4096 juin  17 10:30 .
drwxrwxr-x 10 alex alex 4096 juin  17 10:30 ..
drwxrwxr-x  2 alex alex 4096 juin  17 10:30 destructiveChanges
drwxrwxr-x  3 alex alex 4096 juin  17 10:30 force-app
alex@ubuntu:~/Documents/dentmaster$ ls -la  output/force-app/main/default/
total 12
drwxrwxr-x 3 alex alex 4096 juin  17 10:30 .
drwxrwxr-x 3 alex alex 4096 juin  17 10:30 ..
drwxrwxr-x 2 alex alex 4096 juin  17 10:30 sharingRules
alex@ubuntu:~/Documents/dentmaster$ ls -la  output/force-app/main/default/sharingRules/
total 16
drwxrwxr-x 2 alex alex 4096 juin  17 10:30 .
drwxrwxr-x 3 alex alex 4096 juin  17 10:30 ..
-rw-rw-r-- 1 alex alex 4581 juin  17 10:30 Account.sharingRules-meta.xml
alex@ubuntu:~/Documents/dentmaster$ cat  output/force-app/main/default/sharingRules/
cat: output/force-app/main/default/sharingRules/: Is a directory
alex@ubuntu:~/Documents/dentmaster$ cat  output/force-app/main/default/sharingRules/Account.sharingRules-meta.xml 
<?xml version="1.0" encoding="utf-8"?>
<SharingRules xmlns="http://soap.sforce.com/2006/04/metadata">
    <sharingCriteriaRules>
        <fullName>DWIN2_AccountRH_Edit</fullName>
        <accessLevel>Edit</accessLevel>
        <accountSettings>
            <caseAccessLevel>Read</caseAccessLevel>
            <contactAccessLevel>Edit</contactAccessLevel>
            <opportunityAccessLevel>Read</opportunityAccessLevel>
        </accountSettings>
        <label>DWIN2_AccountRH_Edit</label>
        <sharedTo>
            <roleAndSubordinatesInternal>DWIN2_RH_Rsp</roleAndSubordinatesInternal>
        </sharedTo>
        <criteriaItems>
            <field>RecordTypeId</field>
            <operation>equals</operation>
            <value>RH</value>
        </criteriaItems>
    </sharingCriteriaRules>
    <sharingCriteriaRules>
        <fullName>DWIN2_Account_ADV_Edit</fullName>
        <accessLevel>Edit</accessLevel>
        <accountSettings>
            <caseAccessLevel>Read</caseAccessLevel>
            <contactAccessLevel>Edit</contactAccessLevel>
            <opportunityAccessLevel>Read</opportunityAccessLevel>
        </accountSettings>
        <description>ADV and SC can edit all customer account</description>
        <label>DWIN2_Account_ADV_Edit</label>
        <sharedTo>
            <group>DWIN2_ADV</group>
        </sharedTo>
        <criteriaItems>
            <field>RecordTypeId</field>
            <operation>notEqual</operation>
            <value>RH</value>
        </criteriaItems>
    </sharingCriteriaRules>
    <sharingCriteriaRules>
        <fullName>DWIN2_Account_Client_Read</fullName>
        <accessLevel>Edit</accessLevel>
        <accountSettings>
            <caseAccessLevel>Read</caseAccessLevel>
            <contactAccessLevel>Edit</contactAccessLevel>
            <opportunityAccessLevel>Read</opportunityAccessLevel>
        </accountSettings>
        <description>Tous les comptes clients sont visible par la branche commerciale</description>
        <label>DWIN2_Account_Client_Read</label>
        <sharedTo>
            <roleAndSubordinatesInternal>DWIN2_Direction</roleAndSubordinatesInternal>
        </sharedTo>
        <criteriaItems>
            <field>RecordTypeId</field>
            <operation>notEqual</operation>
            <value>RH,PARTICULIER</value>
        </criteriaItems>
    </sharingCriteriaRules>
    <sharingCriteriaRules>
        <fullName>DWIN2_Account_RH_Read</fullName>
        <accessLevel>Read</accessLevel>
        <accountSettings>
            <caseAccessLevel>None</caseAccessLevel>
            <contactAccessLevel>Read</contactAccessLevel>
            <opportunityAccessLevel>None</opportunityAccessLevel>
        </accountSettings>
        <description>Can view collaborator</description>
        <label>DWIN2_Account_RH_Read</label>
        <sharedTo>
            <role>DWIN2_Direction</role>
        </sharedTo>
        <criteriaItems>
            <field>RecordTypeId</field>
            <operation>equals</operation>
            <value>RH</value>
        </criteriaItems>
    </sharingCriteriaRules>
    <sharingCriteriaRules>
        <fullName>DWIN2_Account_SC_Edit</fullName>
        <accessLevel>Edit</accessLevel>
        <accountSettings>
            <caseAccessLevel>None</caseAccessLevel>
            <contactAccessLevel>Edit</contactAccessLevel>
            <opportunityAccessLevel>None</opportunityAccessLevel>
        </accountSettings>
        <label>DWIN2_Account_SC_Edit</label>
        <sharedTo>
            <group>DWIN2_ServiceClientGroup</group>
        </sharedTo>
        <criteriaItems>
            <field>RecordTypeId</field>
            <operation>notEqual</operation>
            <value>RH</value>
        </criteriaItems>
    </sharingCriteriaRules>
    <sharingCriteriaRules>
        <fullName>DWIN2_SR_Account_Contact_Create</fullName>
        <accessLevel>Edit</accessLevel>
        <accountSettings>
            <caseAccessLevel>None</caseAccessLevel>
            <contactAccessLevel>Edit</contactAccessLevel>
            <opportunityAccessLevel>None</opportunityAccessLevel>
        </accountSettings>
        <description>Can create contact commercial</description>
        <label>DWIN2_SR_Account_Contact_Create</label>
        <sharedTo>
            <group>DWIN2_PartnerCommunity_ROAD</group>
        </sharedTo>
        <booleanFilter>1</booleanFilter>
        <criteriaItems>
            <field>RecordTypeId</field>
            <operation>notEqual</operation>
            <value>RH</value>
        </criteriaItems>
    </sharingCriteriaRules>
</SharingRules>
alex@ubuntu:~/Documents/dentmaster$ cat output/destructiveChanges/package.xml 
<?xml version="1.0" encoding="UTF-8"?>
<Package xmlns="http://soap.sforce.com/2006/04/metadata">
    <version>48.0</version>
</Package>
```

**Comment by scolladon** [comment]
Good catch @alex9138 ! 
It seems to be a linter artefact.
Last commit should fix that.
Could you retest it and tell the result please ?

**Comment by alex9138** [comment]
@scolladon alright here !

```
alex@ubuntu:~/Documents/dentmaster$ ls -la output/
total 20
drwxrwxr-x  5 alex alex 4096 juin  17 11:28 .
drwxrwxr-x 10 alex alex 4096 juin  17 11:28 ..
drwxrwxr-x  2 alex alex 4096 juin  17 11:28 destructiveChanges
drwxrwxr-x  3 alex alex 4096 juin  17 11:28 force-app
drwxrwxr-x  2 alex alex 4096 juin  17 11:28 package
alex@ubuntu:~/Documents/dentmaster$ ls -la output/package/
total 12
drwxrwxr-x 2 alex alex 4096 juin  17 11:28 .
drwxrwxr-x 5 alex alex 4096 juin  17 11:28 ..
-rw-rw-r-- 1 alex alex  557 juin  17 11:28 package.xml
alex@ubuntu:~/Documents/dentmaster$ cat output/package/package.xml 
<?xml version="1.0" encoding="UTF-8"?>
<Package xmlns="http://soap.sforce.com/2006/04/metadata">
    <types>
        <members>Account.DWIN2_AccountRH_Edit</members>
        <members>Account.DWIN2_Account_ADV_Edit</members>
        <members>Account.DWIN2_Account_Client_Read</members>
        <members>Account.DWIN2_Account_RH_Read</members>
        <members>Account.DWIN2_Account_SC_Edit</members>
        <members>Account.DWIN2_SR_Account_Contact_Create</members>
        <name>SharingCriteriaRule</name>
    </types>
    <version>48.0</version>

```

**Comment by scolladon** [comment]
Fix with new version [3.1.1](https://www.npmjs.com/package/sfdx-git-delta)
